### DISCUSSION TITLE
A few recommendations for 1.12.2

### DISCUSSION THREAD

**Discussion by jacket430** [TheUsefulLists/UsefulMods#43] [Ideas]
Hey there! I thought of a few things missing from the 1.12.2 bug fix list, AttributeFix and Born in a Barn. I'm not sure if AttributeFix would be considered a bug fix, but Born in a Barn definitely would be.

**Comment by GeefireMC** [comment]
I think AttributeFix would qualify as a bugfixing mod, even though it fixes bugs with other mods. This repo does include things like DrawerFPS, which is a mod which only helps when other mods are present, so I think AttributeFix is fine here. I've added both mods to the lists.
